sdk/healthinsights/health-insights-clinicalmatching-rest/CHANGELOG.md

```diff
@@ -2,4 +2,6 @@
 
 ## 1.0.0-beta.1 (2023-08-24)
```

Comment:
Are these dates correct?

Reply:
I did not change these dates, so should be ok. Not sure, since I only work on radiologyinsights. Needed to change to be compliant to the new format of the Changelog.